mmocr/models/textrecog/plugins/common_plugins.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
This file is better to be renamed as `common.py`. The shorter name still clearly demonstrates its content to users since it's already under the path `plugins`.